library_generation/repo-level-postprocess/apply_current_versions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script sets the "current-version" written in versions.txt applied to all
```

Comment:
This script is available in google-cloud-java. When you generate a new client library there, you have access to this script.

Reply:
I plan to migrate these files as well since they can be used in repo-level post processing.

Reply:
> in repo-level post processing.

Does this include other repositories than google-cloud-java? If so that makes sense.

Reply:
> Does this include other repositories than google-cloud-java? If so that makes sense.

Yes, these scripts (in `repo-level-postprocess`) are applied to all java modules in google-cloud-java.

Reply:
apply_current_versions.sh is not needed any more?

Reply:
> Does this include other repositories than google-cloud-java?
>  all java modules in google-cloud-java

If the scope is within google-cloud-java, there's no point to copy this to sdk-platform-java.

Reply:
This script is no longer needed.